docs/utilities/recipes.md

```diff
@@ -0,0 +1,209 @@
+Recipes
+=======
+
+With the update to Minecraft 1.12, Mojang introduced a new data-driven recipe system based on JSON files. Since then it has been adopted by Forge as well and will be expanded with Minecraft 1.13.
+All recipe json have to be saved with the UTF-8 encoding.
+
+Loading Recipes
+---------------
+Forge will load all recipes which can be found within the `./assets/<modid>/models/recipes/` folder. You can call these files whatever you want, thought the vanilla convention is to name them after the output item. This name is also used as the registration key, but does not affect the operation of the recipe.
```

Comment:
There are some conditions: see https://github.com/MinecraftForge/MinecraftForge/blob/9cda586ebd402df2e0543444329cfc1dd08172ea/src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java#L697.

Reply:
@temp1011 Meaning all files which start with an underscore ( '_' ) or don't have the '.json' file extension will be ignored. Correct?

Reply:
Yes, it's quite minor though.

Reply:
I just added a small note.